mos_tests/conftest.py

```diff
@@ -269,14 +269,40 @@ def is_l3_ha(env):
 
 @pytest.fixture(autouse=True)
 def env_requirements(request, env):
+    reserved = {'or', 'and', 'not', '(', ')'}
     marker = request.node.get_marker('check_env_')
-    if marker:
-        for func_name in marker.args:
-            func = globals().get(func_name)
-            if func is not None and not func(env):
-                doc = func.__doc__ or 'Env {}'.format(
-                    func_name.replace('_', ' '))
-                pytest.skip('Requires: {}'.format(doc))
+    if not marker:
+        return
+    marker_str = marker.args[0]
```

Comment:
But now there is not code like this in project, if we want to add some of things like this we don't need to add 2 decorators, we can make one condition instead.
Or, could you please get some examples, when this code not allow to do, but that you can do with 2 decorators.